lib/ipam/ipam_block_reader_writer_test.go

```diff
@@ -130,6 +130,11 @@ func (c *fakeClient) Clean() error {
 	return nil
 }
 
+func (c *fakeClient) Close() error {
```

Comment:
Do we actually need this? I don't think this is required for the client interface.

Reply:
No. But seems it is there in private.